### ISSUE TITLE
Fire TV Stick 4K libss-local exits too fast (exit code: 255)

### ISSUE THREAD

**Issue by kevev** [shadowsocks/shadowsocks-android#2461]
Please read contributing guidelines. Thanks.

**Describe the bug**

**To Reproduce**
1. Install shadowsocks--universal-5.0.5.apk
2.) Attempt to connect.

**Expected behavior**
Connect to Shadowsocks server.

**Screenshots**

**Smartphone (please complete the following information):**
 - Android/Chrome OS version: Fire OS 6.2.6.8 (NS6268/2315)
 - Device: Amazon Fire Stick 4K
 - Version: 5.0.5
 - Last version that did not exhibit the issue: 4.7.0 (No error, on connect, but also no internet access when connected.

**Configuration**
Put an `x` inside the [ ] that applies.

* [X ] IPv4 server address
* [ ] IPv6 server address
* [ X] Client IPv4 availability
* [ ] Client IPv6 availability
* Encrypt method: AES-256-CFB
* Route
  * [ X] All
  * [ ] Bypass LAN
  * [ ] Bypass China
  * [ ] Bypass LAN & China
  * [ ] GFW List
  * [ ] China List
  * [ ] Custom rules
* [ X] IPv6 route
* [ ] Apps VPN mode
  * [ ] Bypass mode
* Remote DNS: dns.google
* [ ] DNS over UDP
* Plugin configuration (if applicable):
* [ X] Auto Connect
* [ ] TCP Fast Open
* If you're not using VPN mode, please supply more details here:

**Additional context**

**Comment by kevev** [comment]
This issue resolved itself afterb3 days. Very weird.

**Comment by samirmh-dev** [comment]
still same issue exists+

**Comment by kevev** [comment]
Correct, the issue still exists. After a few hours the client is able to connect. This issue happened whenever the Fire OS device looses wifi connection and then re-establishes connection. After an hour or so the client is finally able to connect to vpn.  Are there any troubleshooting or log gathering steps available?

**Comment by kevev** [comment]
There is another open bug on this. The issue is caused by proxy port 1080 being used by another process. Changing the port number to 1081 for example resolves this issue. The app really should give better error info so the users will know that switching ports is the fix.

**Comment by javad-vovin** [comment]
> There is another open bug on this. The issue is caused by proxy port 1080 being used by another process. Changing the port number to 1081 for example resolves this issue. The app really should give better error info so the users will know that switching ports is the fix.

Thanks, Changing port number fixed the issue
